src/app/components/TransactionsTable/index.tsx

```diff
@@ -97,6 +123,17 @@ export default function TransactionsTable({ transactions }: Props) {
                       ) : (
                         ""
                       )}
+                      <dl>
+                        <dt className="mt-4 font-medium text-gray-800 dark:text-white">
+                          Metadata
```

Comment:
It's not valid/semantic html if a `dt` doesn't have `dd`. Maybe use a `div` with a headline `p` and only wrap the fragment into a `dl`

Reply:
+1